src/compiler/scala/tools/nsc/backend/jvm/opt/BoxUnbox.scala

```diff
@@ -0,0 +1,907 @@
+/* NSC -- new Scala compiler
```

Comment:
No need to get this class in since we don't use optimizations, don't you think?

Reply:
Removed